docs/source/transports.rst

```diff
@@ -0,0 +1,47 @@
+.. _transports_details:
+
+
+Transports
+==========
+
+Transports are used by Solar to communicate with managed nodes.
+Transports are also resuorces, so they have all features of it.
+Transports should be added to a node, but if you need you can add different transports for different resources.
+
+
+How it works
+------------
+
+* Each resource in solar have randomly generated :ref:`transports-id-term`, when you connect resources together,
```

Comment:
Each resource in solar has randmo transport-id generated when resources are connected to each other